Champion kick off (#482)
ropensci/roweb3: content/blog/2023-02-17-champions-kick-off/index.md

---
title: rOpenSci Champions Program Kick off
author: 
  - Yanina Bellini Saibene

> date: '2023-01-17T00:00:01-07:00'

Reviewer: ```suggestion
date: '2023-02-17'
```

Reviewer: I updated the suggested changes to have the times back in

Reviewer: But the times are the same... should they be?

Author: Thank you!

Author: No, no, they should also be different.

Author: I leave that detail until the last moment when I change the folder name and date (in case we have to do it). But the time could be something that is always the same for the Spanish blog post and the English blog post.  And we should add some instruction on the blog guide (if we didn't do it already)

slug: champions-program-kick-off
tags:
  - community
  - champions
feature: true  
# description: 
tweet: "" 
---

> ## The champions program got off to a great start in 2023

Reviewer: ```suggestion
## The Champions Program got off to a great start in 2023!
```

> From September to November 2022, we [launched the rOpenSci Champions Program](/champions/).

Reviewer: ```suggestion
We're happy to report on the first couple of months in our first run of the [rOpenSci Champions Program](/champions/). In September 2022, we launched the program, advertising for both mentors and mentees to apply.
```

We received a total of **102** applications from **31** different countries. Seventy-four applications were for champions and twenty eight for mentors. We are very grateful to everyone who took the time to apply to our program.

Champions could select one of three project options: **51%** chose to **develop a new package**, **25%** to be part of our **review process**, and **24%** to **submit a package to our peer review** process.

> {{< figure src="countriesApplications.png" alt="Map of the world with the countries colored according to the number of applications. Applications are shown in America, Europe, Oceania, Asia and Africa (in that order with respect to the number of applications)." >}}

Reviewer: I don't think this is an issue, but from this map, it's easy for applicants to see that they were the only applicant from their country. I don't *think* that's an issue, but it stuck with me a bit, so I thought I'd mention that and see what you think.

Author: Thank you for pointing this! I also don't think this is an issue.  

Thinking about the people who submitted an application, we are not releasing any data about them, so unless they comment themselves, no one knows who applied. 

Thinking about us, I like to be transparent with this information.  The map shows us that we have to do better in Africa, for example.

> After carefully reviewing each application, we selected [ten champions and ten mentors](/champions/) for this first cohort.  **Three** of our champions will **develop a new package**, **four** will **prepare and submit their package to the review process**, and **three** will **become rOpenSci reviewers**.

Reviewer: I don't think this link has much context here.
```suggestion
After carefully reviewing each application, we selected ten champions and ten mentors for this first cohort.  **Three** of our champions will **develop a new package**, **four** will **prepare and submit their package to the review process**, and **three** will **become rOpenSci reviewers**.
```

Author: I tried to link to the pictures with the data, but but cannot do so.  I think with the photo collage it's more than fine.   Agree with your comment.

The mentors will accompany their mentees throughout the process of executing their projects.

These first participants are from **Argentina, Brazil, Peru, Nigeria, Turkey, Australia, France, Norway, Saudi Arabia, the United States, and the United Kingdom**.

> <FIXME: add collage with champions and mentors pictures>

Reviewer: Fix? (Just a reminder 😁 )

Author: hihi, thanks!!

## Kick-off meetings

> As scheduled, during January, we held the kick-off meetings for the program. Due to time zone differences, we have at least two versions of all planned activities.

Reviewer: ```suggestion
During January, we held the kick-off meetings for the program. Because we our Champions and Mentors have such a wide geographic spread, we had (and will be having) at least two different time for all planned activities to accommodate the breadth of timezones.
```

> During these meetings, mentors and champions introduced themselves and shared where they are from, what your project is about, and something that excites them about the program.   We also went over the logistics of the program, the potential agenda, and scheduled activities.  In these meetings, we noticed the participants' enthusiasm to learn, share, execute their projects, and give back to the community.

Reviewer: ```suggestion
During these meetings, mentors and champions introduced themselves, shared where they are from, what their project is about, and something that excited them about the program. We also went over logistics, the potential agenda, and scheduled activities.  We were excited to see the participants' enthusiasm to learn, share, execute their projects, and give back to the community!
```

## Training

During February and March, champions and mentors will receive training in different topics that will provide them with useful tools to carry out their projects.

> In April and May, we will conduct workshops for those not selected to be champions. These workshops will teach tools for open-source software development.  We hope that this offer can serve to gain skills for their tasks as research software developers and help them with future applications to this or other similar programs.  We will be publishing details about these workshops in March.

Reviewer: ```suggestion
We're also happy to note that in April and May, we will conduct workshops for those not selected to be champions. These workshops will provide an opportunity for all applicants to develop skills in open-source software development for supporting career opportunities as well as future applications to this or other similar programs.  We will be publishing details about these workshops in March.
```
